pkg/infra/tracing/opentelemetry_tracing.go

```diff
@@ -0,0 +1,146 @@
+package tracing
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/grafana/grafana/pkg/infra/log"
+	"github.com/grafana/grafana/pkg/setting"
+	opentracing "github.com/opentracing/opentracing-go"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/exporters/jaeger"
+	"go.opentelemetry.io/otel/sdk/resource"
+	tracesdk "go.opentelemetry.io/otel/sdk/trace"
+	semconv "go.opentelemetry.io/otel/semconv/v1.4.0"
+	trace "go.opentelemetry.io/otel/trace"
+)
+
+type Tracer interface {
+	Start(ctx context.Context, spanName string, opts ...trace.SpanStartOption) (context.Context, Span)
+	Run(context.Context) error
+}
+
+type Span interface {
+	End()
+	SetAttributes(kv ...attribute.KeyValue)
+	// AddEvent(name string, options ...trace.EventOption)
```

Comment:
Probably could be removed

Reply:
yeah sure, on this PR but I need to add it eventually, on the next one